.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ 11 ]
+        java: [ 17 ]
```

Comment:
Just for my curiosity... why does `build.yml` use two Java versions (17, 21) while nightly and release use only 17?

Reply:
We only release one version of the Docker image (and it currently has JDK 17 in it). We could release two Docker images, but I don't see much value in that(?) since we only need one. The 21 is in the regular build.yml to make sure that it's forward-compatible (i.e., that we get a heads-up about something that will potentially break in the future).

The nightly is using 17 because that's the JDK version that becomes our official release. Its main purpose is to be that canary in a cage to indicate whether we have any dependency/image issues. However, this may be less essential now that we've configured this project to use Dependabot. I don't know that we have any other needs for a nightly build.